src/components/WardsTable/index.js

```diff
@@ -1,58 +1,54 @@
 import React from "react";
-import Router from "next/router";
+import AnchorLink from "../AnchorLink";
 
-const WardsTable = ({ wards }) => (
+const WardsTable = ({ wards, wardVisitTotals }) => (
   <div className="nhsuk-table-responsive">
     <table className="nhsuk-table">
       <caption className="nhsuk-table__caption">List of wards</caption>
       <thead className="nhsuk-table__head">
         <tr className="nhsuk-table__row">
-          <th className="nhsuk-table__header" scope="col">
-            Hospital name
-          </th>
+          {!wardVisitTotals && (
+            <th className="nhsuk-table__header" scope="col">
+              Hospital name
+            </th>
+          )}
           <th className="nhsuk-table__header" scope="col">
             Ward name
           </th>
           <th className="nhsuk-table__header" scope="col">
             Ward code
           </th>
+          {wardVisitTotals && (
+            <th className="nhsuk-table__header" scope="col">
+              Booked visits
+            </th>
+          )}
           <th className="nhsuk-table__header" scope="col"></th>
           <th className="nhsuk-table__header" scope="col"></th>
         </tr>
       </thead>
       <tbody className="nhsuk-table__body">
         {wards.map((ward) => (
           <tr key={ward.callId} className="nhsuk-table__row">
-            <td className="nhsuk-table__cell">{ward.hospitalName}</td>
+            {!wardVisitTotals && (
+              <td className="nhsuk-table__cell">{ward.hospitalName}</td>
+            )}
             <td className="nhsuk-table__cell">{ward.name}</td>
             <td className="nhsuk-table__cell">{ward.code}</td>
+            {wardVisitTotals && (
+              <td className="nhsuk-table__cell">{wardVisitTotals[ward.id]}</td>
```

Comment:
```suggestion
              <td className="nhsuk-table__cell">{wardVisitTotals[ward.id] || 0 }</td>
```

Or this 0 handled somewhere else as you get this if a ward has no booked visits / no row in the `wardVisitTotals` table.

![image](https://user-images.githubusercontent.com/42817036/83737449-eb147a80-a64a-11ea-9c7a-f24fd20bde7d.png)

Reply:
Good point. I'll update the usecase to ensure that wards with no visits are still returned and set the total to 0